cypress/integration/errors.js

```diff
@@ -0,0 +1,64 @@
+describe('Error page', () => {
+
+    it('Should handle 404 Errors', () => {
+        cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/', { statusCode: 404 })
+
+        cy.visit('http://localhost:3000')
+
+        cy.get('h2').should('have.text', 'Sorry you smell like the rancidest of the Rancid Tomatillos. Try again or smell you later.')
+
+        cy.get('button')
+            .should('have.text', 'Back to Main')
+            .should('have.css', 'transform')
+
+        cy.get('button').click()
+    })
+
+    it('Should handle 500 Errors', () => {
+        cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/', { statusCode: 500 })
+
+        cy.visit('http://localhost:3000')
+
+        cy.get('h2').should('have.text', 'Sorry you smell like the rancidest of the Rancid Tomatillos. Try again or smell you later.')
+
+        cy.get('button')
+            .should('have.text', 'Back to Main')
+            .should('have.css', 'transform')
+
+        cy.get('button').click()
+    })
+
+    it('Should handle 404 Errors', () => {
+        cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/*', { statusCode: 404 })
+
+        cy.visit('http://localhost:3000/694919')
+
+        cy.get('h2').should('have.text', 'Sorry you smell like the rancidest of the Rancid Tomatillos. Try again or smell you later.')
+
+        cy.get('button')
+            .should('have.text', 'Back to Main')
+            .should('have.css', 'transform')
+
+        cy.get('button').click()
+    })
+
+    it('Should handle 500 Errors', () => {
+        cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/*', { statusCode: 500 })
+
+        cy.visit('http://localhost:3000/694919')
+
+        cy.get('h2').should('have.text', 'Sorry you smell like the rancidest of the Rancid Tomatillos. Try again or smell you later.')
+
+        cy.get('button')
+            .should('have.text', 'Back to Main')
+            .should('have.css', 'transform')
+
+    })
+
+    it('Should be able to click back to Home page', () => {
```

Comment:
Love this test that makes sure we can navigate and tests to make sure we landed where we should.